packages/strapi-design-system/src/themes/colors.js

```diff
@@ -13,7 +13,6 @@ export const colorTokenObject = {
     neutral0: '#ffffff',
     neutral100: '#f6f6f9',
     neutral150: '#eaeaef',
-    neutral200: '#dcdce4',
```

Comment:
@maevalienard we can't remove this one since it's used somewhere in the codebase